src/backend/marsha/bbb/tests/api/classroom/test_retrieve.py

```diff
@@ -158,3 +166,68 @@ def test_api_classroom_fetch_instructor(self, mock_get_meeting_infos):
             },
             content,
         )
+
+    @mock.patch.object(serializers, "get_meeting_infos")
+    def test_api_classroom_fetch_user_access_token(self, mock_get_meeting_infos):
+        """A user with UserAccessToken should not be able to fetch a classroom."""
+        organization = OrganizationFactory()
+        user = UserFactory()
+        OrganizationAccessFactory(organization=organization, user=user)
```

Comment:
Should we update `OrganizationAccessFactory` to create organization and user ?

``` python
class OrganizationAccessFactory(DjangoModelFactory):
    """Factory for the OrganizationAccess model."""

    organization = factory.SubFactory(OrganizationFactory)
    user = factory.SubFactory(ConsumerSiteFactory)

    class Meta:  # noqa
        model = models.OrganizationAccess
```

so we can write 

```suggestion
        organization_access = OrganizationAccessFactory(organization=organization, user=user)
```

then use `organization_access.organization` and `organization_access.user` ? Or it is overkill ?